contracts/staking/contracts/src/staking_pools/MixinCumulativeRewards.sol

```diff
@@ -95,33 +74,19 @@ contract MixinCumulativeRewards is
         // Never set the most recent reward epoch to one in the future, because
         // it may get removed if there are no more dependencies on it.
         if (epoch <= currentEpoch) {
-            _trySetMostRecentCumulativeRewardEpoch(poolId, epoch);
-        }
-    }
+            // Check if we should do any work
+            uint256 currentMostRecentEpoch = _cumulativeRewardsByPoolLastStored[poolId];
+            if (epoch == currentMostRecentEpoch) {
+                return;
+            }
 
-    /// @dev Tries to set the epoch of the most recent cumulative reward.
-    ///      The value will only be set if the input epoch is greater than the
-    ///     current most recent value.
-    /// @param poolId Unique Id of pool.
-    /// @param epoch Epoch of the most recent cumulative reward.
-    function _trySetMostRecentCumulativeRewardEpoch(
-        bytes32 poolId,
-        uint256 epoch
-    )
-        internal
-    {
-        // Check if we should do any work
-        uint256 currentMostRecentEpoch = _cumulativeRewardsByPoolLastStored[poolId];
-        if (epoch == currentMostRecentEpoch) {
-            return;
+            // Update state to reflect the most recent cumulative reward
+            _forceSetMostRecentCumulativeRewardEpoch(
```

Comment:
Does anyone else call this function? It might be better just to inline it here because it's so trivial that it might be clearer.

Reply:
I didn't inline it because it broke too many tests, but I would be down for doing so.